src/community/advanced-data-card.tsx

```diff
@@ -57,46 +64,65 @@ const CardContent: React.FC<CardContentProps> = ({
     subtitleLinesMax,
     description,
     descriptionLinesMax,
+    href,
+    to,
+    ariaLabel,
 }) => {
     const {textPresets} = useTheme();
 
     return (
         <Stack space={4}>
             {headline}
-            <Stack space={4}>
-                {pretitle && (
-                    <Text2
-                        color={vars.colors.textPrimary}
-                        truncate={pretitleLinesMax}
-                        as={pretitleAs}
-                        regular
-                        hyphens="auto"
-                    >
-                        {pretitle}
-                    </Text2>
-                )}
-                <Text
-                    mobileSize={18}
-                    mobileLineHeight="24px"
-                    desktopSize={20}
-                    desktopLineHeight="28px"
-                    truncate={titleLinesMax}
-                    weight={textPresets.cardTitle.weight}
-                    as={titleAs}
-                    hyphens="auto"
+            {href ? (
+                <Touchable
+                    tabIndex={0}
+                    maybe
+                    className={classNames(styles.touchableArea)}
+                    href={href}
+                    to={to}
+                    aria-label={ariaLabel}
+                >
+                    <div />
+                </Touchable>
+            ) : (
+                <Touchable
+                    tabIndex={0}
+                    maybe
+                    className={classNames(styles.touchableArea)}
+                    onPress={onPress}
+                    to={to}
+                    aria-label={ariaLabel}
                 >
-                    {title}
-                </Text>
+                    <div />
+                </Touchable>
+            )}
```

Comment:
you can simplify this by doing something like this: https://github.com/Telefonica/mistica-web/blob/master/src/card.tsx#L511

![image](https://github.com/Telefonica/mistica-web/assets/1849576/b03d7c03-13e1-45f6-bb20-a18696f1da93)